chromium_src/chrome/browser/chrome_browser_main_mac.mm

```diff
@@ -0,0 +1,10 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/chrome_browser_main_mac.h"
+#import "brave/browser/brave_app_controller_mac.h"
+
+#define AppController BraveAppController
```

Comment:
Needs a matching #undef for completeness.

Reply:
done